packages/odyssey-storybook/src/utilities/Utilities.stories.mdx

```diff
@@ -0,0 +1,202 @@
+
+```html
+<Text as="blockquote">
+  <Text as="p">
+    The important thing is not to stop questioning. Curiosity has its own reason
+    for existence.
+  </Text>
+  <footer>
+    Albert Einstein,
+    <Text as="cite">
+      Old Man's Advice to Youth: "Never Lose a Holy Curiosity," LIFE magazine (2
+      May 1955) statement to William Miller, p. 64.
+    </Text>
+  </footer>
+</Text>
+```
+
+####
+
+**WIP API NOTE:** This next section is an additional use-case for `<Text />`.
+
+In addition, `<Text />` provides a simpler API with indirect descendant selectors for natural HTML5 composition or styling dynamic content (e.g. from markdown).
+
+```html
+<Text>
+  <blockquote>
+    <p>
+      The important thing is not to stop questioning. Curiosity has its own
+      reason for existence.
+    </p>
+    <footer>
+      Albert Einstein,
+      <cite>
+        Old Man's Advice to Youth: "Never Lose a Holy Curiosity," LIFE magazine
+        (2 May 1955) statement to William Miller, p. 64.
+      </cite>
+    </footer>
+  </blockquote>
+</Text>
+```
+
+## Current Cases
+
+Currently, we have a few open PRs that stand as practical examples for application of these utilities. Below, I've outlined my current understanding of what each component should look like, given these utilities.
+
+### Field
+
+```html
+// Field
+
+<Box as="{tag}" className="{styles.root}">
+  <Field.Label
+    inputId="{inputId}"
+    required="{required}"
+    optionalLabel="{optionalLabel}"
+    labelHidden="{labelHidden}"
+    as="{TagLabel}"
+  >
+    {label}
+  </Field.Label>
+  {hint && <Field.Hint id="{inputId}">{hint}</Field.Hint>} {children} {error &&
+  <Field.Error id="{inputId}">{error}</Field.Error>}
+</Box>
+
+// Label //// In the case of these label elements, we can treat our Field
+styling as the HTML5 default, removing the need for component-specific styling.
+
+<Text as="{labelTag}" htmlFor="{inputId}">
+  {children} {!required && optionalLabel && (
+  <span className="{styles.optionalLabel}" children="{optionalLabel}" />
+  )}
+</Text>
+
+// Hidden Label
+
+<ScreenReaderText>
+  <Text as="{labelTag}" htmlFor="{inputId}">
+    {children} {!required && optionalLabel && (
+    <span className="{styles.optionalLabel}" children="{optionalLabel}" />
+    )}
+  </Text>
+</ScreenReaderText>
+
+// Hint
+
+<Box
+  as="p"
+  className="{styles.hint}"
+  id="{`${id}-hint`}"
+  children="{children}"
+/>
+
+// Error
+
+<Box as="p" className="{styles.error}" id="{`${id}-error`}">{children} /> </Box>
+```
+
+### Heading
+
+Much like `<Text />` is a convenience wrapper on top of `<Box />`, `<Heading />` is built on `<Text />`.
+
+```html
+// Heading
+
+<Text as="{tag}" className="{componentClass}"> {children} </Text>
+```
+
+My assumption here is that all Headings have parity with our desired HTML5 defaults. The `className` would be used to apply a different level of visual hierarchy to a tag (an `h1` with the styles of an `h3`).
+
```

Comment:
we dont' have any plans for `<Text />` to accept `className`

Reply:
👍🏼 this is internal only to align around our in-flight Heading implementation, specifically @andrewberg-okta's request that we add `h*` elements.

In short: I think that's the right solve as long as we preserve a standard way to swap visual/semantic styling.